### PR TITLE
update space configuration url and route

### PR DESCRIPTION
**Pull request by denisdimitrov**
Changes the url for space configuration to `admin/w/worldSlug/s/spaceSlug`

Fixes: https://github.com/sparkletown/internal-sparkle-issues/issues/1500

### CODE REVIEW

File: src/components/molecules/AdminSpaceCard/AdminSpaceCard.tsx [hunk 1]
```diff
@@ -37,7 +41,10 @@ export const AdminSpaceCard: React.FC<AdminSpaceCardProps> = ({ venue }) => {
         <div className={logoClasses} />
         <AdminShowcaseSubTitle>{venue.name}</AdminShowcaseSubTitle>
       </div>
-      <ButtonNG linkTo={adminNGVenueUrl(venue.slug)} disabled={!venue.slug}>
+      <ButtonNG
+        linkTo={adminNGVenueUrl(worldSlug, venue.slug)}
+        disabled={!venue.slug}
```
Comment (goran-peoski-work) [inline src/components/molecules/AdminSpaceCard/AdminSpaceCard.tsx:46]: Maybe something like this?
```suggestion
        disabled={!worldSlug || !venue.slug}
```

File: src/components/organisms/AppRouter/AdminV3Subrouter.tsx [hunk 2]
```diff
@@ -27,6 +27,8 @@ import { AdminVenueView } from "components/organisms/AdminVenueView";
 export const AdminV3Subrouter: React.FC = () => (
   <Switch>
     <Route
+      exact
+      strict
```
Comment (goran-peoski-work) [inline src/components/organisms/AppRouter/AdminV3Subrouter.tsx:31]: [NOTE]

This is resolved with re-arranging of routes in a next PR. Good enough for now.

**Review by goran-peoski-work — APPROVED** [review]
LGTM

A comment left for the `disabled` check since `worldSlug` is defined with `?`

File: src/components/molecules/AdminSpaceCard/AdminSpaceCard.tsx (hunk 1 above)
Comment (denisdimitrov) [reply src/components/molecules/AdminSpaceCard/AdminSpaceCard.tsx]: Fixed in: https://github.com/sparkletown/sparkle/pull/2611/commits/1c9d667fb2f36ca250ca3e965dafc8494c390076

**Pull request merged**